offline/QA/Jet/ConstituentsinJets.cc

```diff
@@ -190,17 +193,17 @@ int ConstituentsinJets::process_event(PHCompositeNode *topNode)
   if (!jets)
   {
     std::cout << "ConstituentsinJets::process_event - Error can not find jet node " << m_recoJetName << std::endl;
-    exit(-1);  // fatal
+    //exit(-1);  // fatal
```

Comment:
I'd suggest instead returning here: that way you safely skip processing the rest of this module but the ROOT process stays intact! (The `EVENT_OK` status ensures that modules regsitered after this are still run.)
```suggestion
    return Fun4AllReturnCodes::EVENT_OK;
```